jobbergate-cli/poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
```

Comment:
I see no changes to `pyproject.toml`, why do we have changes on `poetry.lock`?

Reply:
The previous iteration of the prototype was using some external dependencies (pynput, for instance), but it was removed for the final version.